electron/preload.js

```diff
@@ -26,4 +26,8 @@ contextBridge.exposeInMainWorld('api', {
     const result = await ipcRenderer.invoke('save-output', savePath, extractedData);
     return result;
   },
+  saveConfigAs: async (schema) => {
+    const result = await ipcRenderer.invoke('save-config-as', schema);
+    return result;
```

Comment:
We could probably just return the result of ipcRendered.invoke directly, instead of awaiting this fn call.

Reply:
I modified this return statement, as well as the return statements in the other window.api functions.